src/Language/Fixpoint/Types/Names.hs

```diff
@@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
   , dropSym
   , headSym
   , lengthSym
+  , mappendSym
```

Comment:
I believe we don't export this on purpose. Constructing a `Symbol` involves hashing the string and performing a table lookup, which is expensive, so we want to keep symbol construction to a minimum. Better to perform all formatting operations on the raw `Text` and then make a single `Symbol` at the end.